test/functional/shared.js

```diff
@@ -1,7 +1,8 @@
 'use strict';
 
-var MongoClient = require('../../').MongoClient,
-  expect = require('chai').expect;
+const Promise = require('bluebird');
```

Comment:
Why do we need to pull bluebird in here?

Reply:
yeah left it in from some experimentation, we can remove it. FYI, incoming changes to also remove all the `define` stuff